apps/src/templates/CodeWorkspaceContainer.jsx

```diff
@@ -99,6 +151,18 @@ const styles = {
     borderTopStyle: 'solid',
     borderTopColor: '#ddd',
   },
+  [MODAL_EDITOR_ID]: {
```

Comment:
that is out of scope for this pr since it would require moving all the styles out (if we wanted to do it right) and testing to make sure things work well.